app/src/renderer/js/size-selector.js

```diff
@@ -56,15 +62,49 @@ async function getWindowList() {
       .map(win => {
         const iconImage = images.find(img => img.pid === win.pid);
         const icon = iconImage.icon ? nativeImage.createFromBuffer(iconImage.icon) : null;
-        return Object.assign({}, win, {
+        return {
+          ...win,
           isFullscreen: false,
           icon2x: icon || null,
           icon: icon ? icon.resize({width: 16, height: 16}) : null
-        });
+        };
       })
   ];
 }
 
+function setAppLastUsed(app) {
+  const {count} = usageHistory[app.pid] || {};
+  usageHistory[app.pid] = {
+    count: (Number.isNaN(count) ? 0 : count) + 1,
```

Comment:
When I asked you to change this from `isNaN` I didn't realize that the check was relying on coercion (Also the exact reason `Number.isNaN` exists). This check is now faulty as it will never be `true`. Instead use: `typeof count === 'number' ? ++count : 1`